_posts/2020-10-13-community-call-dei.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Summary of October 2020 Community Call - US-RSE's Role in Promoting Diversity, Equity and Inclusion
+tags: [community call]
+posted_by: Nicole Brewer, Daniel S. Katz, Sandra Gesing, Ian Cosden, Chris Hill
+---
+
+On September 10, 2020, the US-RSE Organization held a community call on "US-RSE's role in promoting Diversity, Equity and Inclusion". The goal of the call was to have an open discussion about what steps US-RSE might take in promoting diversity in our profession, both within the US-RSE organization and externally through outreach to the broader community, and if the participants in the call supported the idea of taking such steps.
+
+We initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.
```

Comment:
The post opens talking about the group in the third person, so it's strange to see "we" here - can we choose one point of view and be consistent?

Reply:
Maybe we change the 2nd paragraph to:
> The call attendees initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.

The problem is there are 2 potential "we"s here -the people on the call and the US-RSE.

Reply:
Perhaps replace "so in essence, we say" with "intimates" or "suggests"
> The call attendees initially talked about the fact that US-RSE's Code of Conduct includes some discussion of diversity, but that our mission does not currently include diversity, which intimates that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.

Reply:
I don't think that addresses the problem

Reply:
I agree it is a bit confusing. There are a lot of we's and I don't mind it (sounds more personal and approachable) What about adding a section header after the first paragraph? So then it has a logical break and transition from the 3rd person in the first paragraph.

```suggestion
### Call Overview & Synopsis

We initially talked about the fact that [US-RSE's Code of Conduct](https://us-rse.org/code-of-conduct/) includes some discussion of diversity, but that our mission does not currently include diversity, so in essence, we say that we are open to and supportive of diversity but we are not actively encouraging it either in our organization or in the RSE profession.
```